<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorState</class>
 <widget class="QWidget" name="GeneratorState">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeneratorState</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditContext"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBoxContext">
     <property name="text">
      <string>context</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="InterfaceEditor" name="interfaceEditor">
     <property name="text">
      <string>define interface</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditState"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBoxInterface">
     <property name="text">
      <string>state interface</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBoxState">
     <property name="text">
      <string>state</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>!!! under construction !!!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InterfaceEditor</class>
   <extends>QPushButton</extends>
   <header>InterfaceEditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
